### PR TITLE
Add thermal model dataset and change name to live script

### PR DESCRIPTION
**Pull request by mfussi66**
As per title. 
xlsx files contain the thermal camera measurements, while .mat files contain the matlab logs taken by three_boards_receiver.slx.

### CODE REVIEW

**Review by pattacini — CHANGES_REQUESTED** [review]
Missing README.

File: boards/amcbldc/resources/project/xjG_2pGCcS41eiFoMEFGWGp5hTo/yyBOcJNGvvXjkAPPZqnaStA2CcMd.xml
Comment (pattacini) [inline boards/amcbldc/resources/project/xjG_2pGCcS41eiFoMEFGWGp5hTo/yyBOcJNGvvXjkAPPZqnaStA2CcMd.xml:1]: This still needs to be reset.

**Comment by mfussi66** [comment]
> Also, is the live script capable of reading the data files from `data`?

Yep, changed the target directory.

File: boards/amcbldc/resources/project/xjG_2pGCcS41eiFoMEFGWGp5hTo/yyBOcJNGvvXjkAPPZqnaStA2CcMd.xml
Comment (mfussi66) [reply boards/amcbldc/resources/project/xjG_2pGCcS41eiFoMEFGWGp5hTo/yyBOcJNGvvXjkAPPZqnaStA2CcMd.xml:1]: Done

**Comment by mfussi66** [comment]
> Missing README.

Added it!

**Review by pattacini — APPROVED** [review]
Thanks heaps!

**Pull request merged**